acl-lib/src/cmr/acl/core.clj

```diff
@@ -167,12 +166,9 @@
            seq))
     (catch Exception e
       (info "Caught exception getting permitting ACLs: " (.getMessage e))
-      (if (re-matches #".*status 401.*" (.getMessage e))
+      (when (re-matches #".*status 401.*" (.getMessage e))
```

Comment:
just keep this one in mind, I'm 99.9% sure that when will return nil but if everything falls apart, I would start looking at one of these.  :)

Reply:
When returns nil if the test is false.  This is the right fix.